integration-tests/kotlin-serialization/src/main/kotlin/io/quarkus/it/kotser/GreetingResource.kt

```diff
@@ -17,6 +17,13 @@ class GreetingResource {
         return Person("Jim Halpert")
     }
 
+    @Path("suspend")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    suspend fun suspendHello(): Person {
```

Comment:
Maybe use a wrapped variant like List<Person>?

Reply:
No need, it's already handled

Reply:
But I did it anyway :)